### ISSUE TITLE
On Maui Android, after picking a bluetooth device, a white screen is overlaid

### ISSUE THREAD

**Issue by euquiq** [inthehand/32feet#289]
Hi!

Same code, was working before latest nuGet update
Consider the following (using  (InTheHand.Net.Bluetooth 4.0.34.1):

```
        BluetoothDeviceInfo device = null;
        var picker = new BluetoothDevicePicker();
        device = await picker.PickSingleDeviceAsync();
```

The bluetooth device picker is shown, I select a device, which is returned on var "device" ....  that part works fine. 
But the app shows a white empty screen, rendered by the BluetoothDevicePicker.

If I revert into  InTheHand.Net.Bluetooth 4.0.33.1, the picker works fine and no white screen is kept on top of my app after selecting the bluetooth device.

Thanks and hopefully this is easy to check / fix.
Enrique.

**Comment by pskofod** [comment]
I have the same problem on 4.0.34.1. Using 4.0.30 no white screen appears.

**Comment by peterfoot** [comment]
This should be resolved in 4.0.36 - the picker has been reworked to remove the in-between activity so the native Bluetooth activity is called directly from the host application.
